### PR TITLE
docs(README): Document action inputs

### PR DESCRIPTION
**Pull request by Kurt-von-Laven**
Enhance input documentation in `action.yaml` as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,6 +216,52 @@ jobs:
...
+
+### Required
+
+#### `bot-token`
+
+The Slack API bot token for your custom app with `chat:write` scope.
+
+#### `channel-id`
+
+The ID of a Slack channel your bot is in to send notifications to.
```
Comment (mwarres) [inline README.md]: The wording here is a bit awkward/confusing...if there is a reasonable way to improve this, that would be excellent

**Review by mwarres — CHANGES_REQUESTED** [review]
Changing to the original two-sentence description would be easier to comprehend

File: README.md (hunk 1 above)
Comment (Kurt-von-Laven) [reply README.md]: Took another stab at it; let me know what you think!

Comment (mwarres) [reply README.md]: Much easier to understand--thanks!

**Review by mwarres — APPROVED** [review]
LGTM

**Pull request merged**